performance_manager/lib/logging_utils.py

```diff
@@ -46,7 +46,7 @@ def log_start(self) -> None:
 
     def log_complete(self) -> None:
         """log the completion of a proccess with duration"""
-        duration = self.start_time - time.time()
+        duration = time.time() - self.start_time
```

Comment:
get rid of those negative durations

Reply:
ha. i had this in my debugging branch as well.